tests/test_schema.py

```diff
@@ -151,5 +168,37 @@ def test_filetest_defaults():
     file_test = FileTest(path="bla")
     assert file_test.contains == []
     assert file_test.must_not_contain == []
+    assert file_test.contains_regex == []
+    assert file_test.must_not_contain_regex == []
     assert file_test.md5sum is None
     assert file_test.should_exist
+
+
+def test_contenttest_with_contains():
+    """ Test if we can make a ContentTest object without regex to match """
+    ContentTest(contains=["Should contain"],
+                must_not_contain=["Should not contain"])
+
+
+def test_contenttest_with_regex():
+    """ Test if we can make a ContentTest object without regex to match """
```

Comment:
I've fixed the docstring. I've kept the test in, even though it's codepath is also used by other test, since it explicitly tests a change in the code itself (namely that `contains_regex` and `must_not_contain_regex` were added to the `__init__` function).